packages/algob/src/lib/account.ts

```diff
@@ -71,3 +71,17 @@ export function mkAccountIndex (accountList: Account[]): Accounts {
   }
   return out;
 }
+
+export function loadFromEnv (): Account[] {
+  var algobAccountsString = process.env.ALGOB_ACCOUNTS;
+  if (algobAccountsString) {
+    const algob = JSON.parse(algobAccountsString);
+    var algobAccounts: Account[] = [];
+    for (const account of algob) {
+      const accountSDK = parseMnemonic(account.mnemonic);
+      algobAccounts.push({ name: account.name, addr: accountSDK.addr, sk: accountSDK.sk });
+    }
+    return algobAccounts;
```

Comment:
this should be in `try-catch` and in case of error, appropriate error should be returned, mentioning which account went wrong:

* mnemonic is missing
* name is missing
* mnemonic parsing failed

Reply:
ok